docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -719,6 +719,63 @@ Then number of elements found by `accessibilityId(slide 2)` is equal to `1`
 ----
 
 
+=== Execute sequence of actions
+
+Executes the sequence of actions
+
+[source,gherkin]
+----
+When I execute sequence of actions: $actions
+----
+* `$actions` - table of actions to execute
+
+==== Possible actions
+
+[cols="1,5,3", options="header"]
+|===
+
+|Type
+|Argument
+|Example
+
+|PRESS
+|Locator
```

Comment:
I think it makes sense to make Locator as a anchor to the locator docs on top of this doc